src/octopeer-github/content/ElementEventBinding/ElementEventBindingData.ts

```diff
@@ -37,11 +37,45 @@ const elementEventBindingDataList: ElementEventBindingData[] = [
         name: "mouseleave",
     },
     {
-        eventID: EventID.SCROLL_INTO_VIEW,
-        name: "scrollintoview",
-    },
-    {
-        eventID: EventID.SCROLL_OUT_OF_VIEW,
-        name: "scrolloutofview",
+        addDOMEvent: (elementSelectionBehaviour: ElementSelectionBehaviour) => {
```

Comment:
@Exclaminator since you're the master of architecture for us, can you take a quick look at this? If there is no easy fix I think we should leave it.